### DISCUSSION TITLE
Did  TensorFlowV2Classifier get migrated somewhere in new 1.8.0 version of ART?

### DISCUSSION THREAD

**Discussion by thungp** [Trusted-AI/adversarial-robustness-toolbox#1336] [Q&A] [answered]
ref:
(a) https://github.com/Trusted-AI/adversarial-robustness-toolbox/discussions/1322

ART: 1.8.0

Getting this error:
---> 16 from art.classifiers import TensorFlowV2Classifier
     17 
     18 import os

ModuleNotFoundError: No module named 'art.classifiers'

I see that the package art.classifiers was removed in 1.8.0
Is there an equivalent class in the new packages that replaced it?

art.preprocessing.expectation_over_transformation, art.estimators.classification and art.estimators.classification.QueryEfficientGradientEstimationClassifier

**Comment by beat-buesser** [comment, marked as answer]
Hi @thungp Yes, `art.classifiers` has been deprecated and removed with ART 1.8.0. Please update imports with `from art.estimators.classification import TensorFlowV2Classifier`, for example.
